images/scripts/fedora.setup

```diff
@@ -220,6 +220,26 @@ systemctl --all --legend=false list-units 'pm*' | awk '{print $1}' | xargs syste
 # logins performed by some tests.
 echo "PerSourcePenalties no" >/etc/ssh/sshd_config.d/99-no-penalties.conf
 
+# HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2374928
+if [ "${IMAGE}" = 'fedora-43' ] || [ "${IMAGE}" = 'fedora-rawhide' ]; then
+    cat <<EOF > hack-ssh-credentials.te
+module hack-ssh-credentials 1.0;
+require {
+    type init_var_run_t;
+    type sshd_t;
+    class file { getattr open read };
+}
+allow sshd_t init_var_run_t:file { getattr open read };
+EOF
+    checkmodule -M -m -o hack-ssh-credentials.mod hack-ssh-credentials.te
+    semodule_package -o hack-ssh-credentials.pp -m hack-ssh-credentials.mod
+    semodule -X 300 -i hack-ssh-credentials.pp
+
+    # This is rwxrwxr-x by default and then we see:
+    #   sshd-session[1582]: Authentication refused: bad ownership or modes for directory /
+    chmod 755 /
```

Comment:
Sorry, no -- this is too intrusive. We want realistic OS images, not some intrusive modifications that happen to make tests work. This smells like it's looking for ssh keys in the root directory or so? Hacking the SELinux policy is already very borderline, as it has repercussions on things that we actually test -- but you could convince me to swallow that if you yell at me loud enough. But this is too much.

Reply:
Even as I was doing this, I felt bad about adding this without so much as a downstream bug reference.  I knew you'd catch me on it :)

I'll look into it and see if I can find out why we end up with such a weird mode here...